[BUG] - Seaborne Communities Failure to Render

**Failure to Render: Communities/Seaborne**
The Seaborne Community in the Compendium will not render more than once. All others work as intended.

Console log reports Failed to render. Log included in additional context.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to Compendium.
2. Click on `Communities` in the Daggerheart SRD/Character Options folder.
3. Click on Seaborne
4. Close template.
5. Click on Seaborne again.

Additional:
If application is closed and restarted, it again appears once.

**Expected behavior**
The Seaborne Community compiled template should appear each time clicked unless already rendered.

**Setup Information:**
 - OS: Windows 11
 - Foundry Desktop App
 - Foundry v13 build 347
 - Foundryheart release 1.0.0


**Additional context**
Console error log if opened from Compendium:
```
foundry.mjs:27195 Uncaught (in promise) Error: Failed to render Application "CommunitySheet-Compendium-daggerheart-communities-Item-o5AA5J05N7EvH1rN":
Failed to render template part "features":
Cannot read properties of null (reading 'getRollData')
    at #render (foundry.mjs:27195:13)
    at async #try (foundry.mjs:7154:17)Caused by: Error: Failed to render template part "features":
Cannot read properties of null (reading 'getRollData')
    at CommunitySheet._renderHTML (foundry.mjs:28563:17)
    at async #render (foundry.mjs:27186:22)
    at async #try (foundry.mjs:7154:17)Caused by: TypeError: Cannot read properties of null (reading 'getRollData')
    at Object.rollParsed (daggerheart.js:23477:55)
    at Object.c (handlebars.min.js:27:27865)
    at eval (eval at createFunctionContext (handlebars.min.js:29:10011), <anonymous>:15:153)
    at Object.h [as fn] (handlebars.min.js:27:25975)
    at Object.<anonymous> (handlebars.min.js:27:18211)
    at Object.c (handlebars.min.js:27:27865)
    at Object.eval [as main] (eval at createFunctionContext (handlebars.min.js:29:10011), <anonymous>:10:50)
    at c (handlebars.min.js:27:23534)
    at d (handlebars.min.js:27:23849)
    at e (handlebars.min.js:28:30886)
```

--- a/module/helpers/handlebarsHelper.mjs
+++ b/module/helpers/handlebarsHelper.mjs
@@ -60,7 +60,7 @@ export default class RegisterHandlebarsHelpers {
 
     static rollParsed(value, actor, item, numerical) {
         const isNumerical = typeof numerical === 'boolean' ? numerical : false;
-        const result = itemAbleRollParse(value, actor.getRollData(), item);
+        const result = itemAbleRollParse(value, actor?.getRollData() ?? {}, item);
         return isNumerical ? (!result ? 0 : Number(result)) : result;
     }
 
@@ -69,7 +69,7 @@ export default class RegisterHandlebarsHelpers {
     }
 
     static empty(object) {
-        if(!(typeof object === 'object')) return true;
+        if (!(typeof object === 'object')) return true;
         return Object.keys(object).length === 0;
     }
 }

